lldb/source/Commands/Options.td

```diff
@@ -1117,6 +1117,11 @@ let Command = "thread plan list" in {
     Desc<"Display thread plans for unreported threads">;
 }
 
+let Command = "thread select" in {
+  def thread_thread_id : Option<"thread_id", "t">,
```

Comment:
Change the long option name to be "thread-id" (replace the '_' with a '-'). That is more consistent with the other commands (see above on line 1113).

Reply:
I don't know about the underlying code, but our command definition interface doesn't allow multiple long options mapping to the same short option.  You could add a long option `--tid` that has no short option if you wanted to provide that.  You don't really get any benefit to having the short option pointing to both long options anyway.  But I don't think we do this anywhere else, and that does seem a little odd to me.